### PR TITLE
Adding 3.8 version support json policy file

### PR DESCRIPTION
**Pull request by Abdiramen**

### CODE REVIEW

File: docs/agent/version-support-policy.mdx [hunk 1]
```diff
@@ -44,6 +44,7 @@ Extended support beyond LTS is available to customers on a commercial basis. It
 
 | Support Status | Release | Release Date | Latest | Active Support | End of Life |
 | -------------- | ------- | ------------ | ------ | -------------- | ----------- |
+| Active         | 3.8x    | 2024-03-14   | 3.7.0  | 2025-03-14     | 2025-05-14  |
```
Comment (bobzilladev) [inline docs/agent/version-support-policy.mdx:47]: Latest to 3.8.0

Comment (Abdiramen) [reply docs/agent/version-support-policy.mdx:47]: ope, thanks!

**Review by salilsub — CHANGES_REQUESTED** [review]
With the "what's new", can we just list which version is being released and a link to the chagelog?

**Comment by bobzilladev** [comment]
@Abdiramen Looks like a change is requested for this changelog PR, and it needs to be deconflicted.

**Comment by Abdiramen** [comment]
> With the "what's new", can we just list which version is being released and a link to the chagelog?

fixed here: https://github.com/ngrok/ngrok-docs/pull/641/commits/1b5f4903fdb2b98d4ceb69a1484ef47fc96767b4

**Review by bobzilladev — APPROVED** [review]

**Pull request merged**